packages/backend/server/utils/markdown.ts

```diff
@@ -114,9 +114,10 @@ _commit: <a href="${checkRunUrl}"><code>${abbreviateCommitHash(workflowData.sha)
 
 function generateTemplatesStr(templates: Record<string, string>) {
   const entries = Object.entries(templates).filter(([k]) => k !== "default");
-  let str = templates.default
-    ? `[Open in Stackblitz](${templates.default})`
-    : "";
+  let str =
```

Comment:
I think the default template is being uploaded anyway! so maybe we should avoid it in that case from the cli's side.